pkg/email/templates/collaborator_changed.html.tmpl

```diff
@@ -0,0 +1,30 @@
+{{- define "title" -}}
+Collaborator Changed
+{{- end -}}
+
+{{- define "preview" -}}
+A collaborator of your {{ .Notification.EntityIds.EntityType }} "{{ .Notification.EntityIds.IDString }}" has been added or updated.
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+Dear {{ .ReceiverName }},
+</p>
+<p>
+A collaborator of your {{ .Notification.EntityIds.EntityType }} "{{ .Notification.EntityIds.IDString }}" on {{ .Network.Name }} has been added or updated.
+</p>
```

Comment:
```suggestion
<p>
Dear {{ .ReceiverName }},<br/><br/>
A collaborator of your {{ .Notification.EntityIds.EntityType }} "{{ .Notification.EntityIds.IDString }}" on {{ .Network.Name }} has been added or updated.<br/><br/>
…
</p>
```
Nit, the overuse of `<p/>` makes scanning quite hard and is also not semantic.

Reply:
Not going to do this. Feel free to rewrite the HTML in a future PR if you feel like it.